cmake/multi_image.cmake

```diff
@@ -158,19 +158,24 @@ function(add_child_image_from_source)
   # | - prj.conf
   # | - prj_<desc>.conf
   # | - child_image DIRECTORY
-  #     |-- <ACI_NAME>.conf (I)             Fragment, used together with (A) and (C)
-  #     |-- <ACI_NAME>_<desc>.conf (J)      Fragment, used together with (B) and (D)
+  #     |-- <ACI_NAME>.conf (I)                 Fragment, used together with (A) and (C)
```

Comment:
This section is included directly in the docs, could you ensure that the docs (multi image docs) still makes sense now that DT overlays are supported as well?

Reply:
I've extended the documentation with new section: `Child image devicetree overlays`